### ISSUE TITLE
Accordion content retains tab stops for many items (Toggle is one example)

### ISSUE THREAD

**Issue by bragusa** [carbon-design-system/carbon#3429]
Accordion content tab stops: closed accordion items may still have tab stops within as content is hidden using visibility rather than display: none. Many HTML elements will honor visibility and be removed from the tab stops, but other will not. Some examples are Carbon Checkbox and Carbon Toggle.

>What did you expect to happen?
It is expected that hidden items are removed from tab order within a page

> What happened instead?
Some items retain their tab index

> What WCAG 2.1 checkpoint does the issue violate?
WCAG 2.4.3

1. Create and accordion with two accordion items
2. Add a Toggle to the first accordion item

At runtime
1. Focus on the first Accordion item title
4. Press Tab
Focus stops on the toggle item

**Comment by SimonFinney** [comment]
Can we revive this issue? We have an AVT item open against this - https://github.com/carbon-design-system/ibm-security/issues/142
